EIPS/eip-4445.md

```diff
@@ -0,0 +1,179 @@
+    /// When a Transfer event emits, this also indicates that the consumer address
+    /// for that NFT (if any) is set to none
+    event ConsumerChanged(address indexed owner, address indexed consumer, uint256 indexed tokenId);
+
+    /// @notice Get the consumer address of an NFT
+    /// @dev The zero address indicates that there is no consumer
+    /// Throws if `_tokenId` is not a valid NFT
+    /// @param _tokenId The NFT to get the consumer address for
+    /// @return The consumer address for this NFT, or the zero address if there is none
+    function consumerOf(uint256 _tokenId) view external returns (address);
+
+    /// @notice Change or reaffirm the consumer address for an NFT
+    /// @dev The zero address indicates there is no consumer address
+    /// Throws unless `msg.sender` is the current NFT owner, an authorised
+    /// operator of the current owner or approved address
+    /// Throws if `_tokenId` is not valid NFT
+    /// @param _consumer The new consumer of the NFT
+    function changeConsumer(address _consumer, uint256 _tokenId) external;
+}
+```
+
+Every contract implementing the `ERC721Consumable` extension is free to define the permissions of a `consumer` (e.g. what
+are consumers allowed to do within their system) with only one exception - consumers MUST NOT be considered owners,
+authorised operators or approved addresses as per the ERC-721 specification. Thus, they MUST NOT be able to execute
+transfers & approvals.
+
+The `consumerOf(uint256 _tokenId)` function MAY be implemented as `pure` or `view`.
+
+The `changeConsumer(address _consumer, uint256 _tokenId)` function MAY be implemented as `public` or `external`.
+
+The `ConsumerChanged` event MUST be emitted when a consumer is changed.
+
+On every `transfer`, the consumer MUST be changed to the zero address the same way `approve` address is changed
+
+## Rationale
+
+Key factors influencing the standard:
+
+- Keeping the number of functions in the interfaces to a minimum to prevent contract bloat.
+- Simplicity
+- Gas Efficiency
+- Not reusing or overloading other already existing roles (e.g. owners, operators, approved addresses)
+
+### Name
+
+The chosen name resonates with the purpose of its existence. Consumers can be considered entities that utilise the token
+instances, without necessarily having ownership rights to it.
+
+The other name for the role that was considered was `operator`, however it is already defined and used within
+the `ERC-721` standard.
+
+### Restriction on the Permissions
+
+There are numerous use-cases where a distinct role for NFTs is required that MUST NOT have owner permissions. A contract
+that implements the consumer role and grants ownership permissions to the consumer renders this standard pointless.
+
+## Backwards Compatibility
+
+There are no other standards that define a similar role for NFTs and the name (`consumer`) is not used by other ERC-721
+related standards.
+
```

Comment:
[EIP-1](https://eips.ethereum.org/EIPS/eip-1#what-belongs-in-a-successful-eip) specifies a "Test Cases" section. You can add tests to the implementation repo and link to them, if you'd like.